content/zh/docs/reference/using-api/health-checks.md

```diff
@@ -51,41 +51,35 @@ This can be useful for a human operator to debug the current status of the Api s
 对于所有端点，都可以使用 `verbose` 参数来打印检查项以及检查状态。
 这对于操作人员调试 API 服务器的当前状态很有用，这些不打算给机器使用：
 
-```shell
-curl -k https://localhost:6443/livez?verbose
-```
+    curl -k https://localhost:6443/livez?verbose
```

Comment:
don't do this!
We prefer explicit code fences to indentation based code blocks.